benchmark/galsim/Sandbox.jl

```diff
@@ -0,0 +1,24 @@
+module Sandbox
```

Comment:
Ah, good to know. Also I didn't mean to check in this file and don't really think it should be checked in. Will remove.